protocol/x/vault/keeper/msg_server_deposit_to_vault_test.go

```diff
@@ -0,0 +1,71 @@
+package keeper_test
+
+import (
+	"errors"
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/dydxprotocol/v4-chain/protocol/mocks"
+	"github.com/dydxprotocol/v4-chain/protocol/testutil/constants"
+	keepertest "github.com/dydxprotocol/v4-chain/protocol/testutil/keeper"
+	"github.com/dydxprotocol/v4-chain/protocol/x/vault/keeper"
+	"github.com/dydxprotocol/v4-chain/protocol/x/vault/types"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMsgServerDepositToVault(t *testing.T) {
```

Comment:
What exactly is this testing? it seems like you're just mocking a function to return an error, and checking that the error is returned when the function is called

Reply:
Yeah this is the pattern with existing `msgServer`s

Reply:
with your below comment on `app_test.go`, i'm going to remove mock testings here and move tests in `app_test.go` to this file